src/main/webapp/js/datahandlers/collegepagedatahandler.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @fileoverview This class handles populating the college page view with data from the servlet. */
+
+goog.module('finscholar.collegepageview.datahandler');
```

Comment:
Is there a reason why you are not matching the namespace to the directory structure?

Reply:
Oh, didn't think about that. Good idea!

Reply:
Hmm, just some thoughts. What if we do the data subsystem the same way we structured the page controllers? So we have a base datahandler module that collegedatahandler and scholarshipdatahandler extend.

Reply:
That works